_pages/connect.md

```diff
@@ -6,9 +6,14 @@ ref: resources
 ---
 
 ### Find a Campus Clubs and a Regional Coordinator near you
+<ul>
+<li>Regional Coordinators are marked with a Yellow Star</li>
+<li>Active Campus Clubs are marked with Red</li>
+<li>Campus Clubs in Training are marked with Blue</li>
+<li>Campus Clubs who have applied but haven't enrolled to the training yet are marked with Gray</li>
+</ul>
+You can also search your University or Regional Coordinator by name if you click on the full screen icon on the top right corner of the map.
 
-#### Campus Clubs are with the red marker and Regional Coordinators with the blue.
+<iframe src="https://www.google.com/maps/d/u/1/embed?mid=1EIAJlq6o94i3o-OQ9jc_OHBf5ZQ" width="1024" height="768"></iframe>
```

Comment:
Hi Nikos, I have removed all the relevant leaflet/tabletop CSS/JS. Please check d5c993b